app/src/docs/v1.api-spec.yaml

```diff
@@ -50,6 +1095,663 @@ components:
+          example: aeb3b705-1de5-4f4e-a4e6-0716b7671034
+        name:
+          type: string
+          example: Business Form One
+        slug:
+          type: string
+          example: businessformone
+        public:
+          type: boolean
+        active:
+          type: boolean
+        keywords:
+          type: array
+          example: [contact, business, form]
+          items:
+            type: string
+        prefix:
+          type: string
+          example: bus_os
+        createdBy:
+          type: string
+          example: migration-004
+        createdAt:
+          type: string
+        updatedBy:
+          type: string
+        updatedAt:
+          type: string
+
+    FormDefinition:
+      type: object
+      properties:
+        formId:
+          type: string
+          example: d6944dd9-1c0c-4915-971d-542dda14eb7f
+        description:
+          type: string
+          example: This is an attestation form for Agriculture and Seafood
+        createdBy:
+          type: string
+          example: migration-011
+        createdAt:
+          type: string
+          example: 2020-06-04T18:49:20.672Z
+        updatedBy:
+          type: string
+          example: migration-012
+        updatedAt:
+          type: string
+          example: 2020-06-04T18:49:20.672Z
+        metadata:
+          type: object
+          $ref: '#/components/schemas/Form'
+        versions:
+          type: array
+          items:
+            type: object
+            $ref: '#/components/schemas/FormVersion'
+
+    FormVersion:
+      type: object
+      properties:
+        formVersionId:
+          type: string
+          example: 1
+        formId:
+          type: string
+          example: d6944dd9-1c0c-4915-971d-542dda14eb7f
+        changes:
+          type: string
+          example: null
+        createdBy:
+          type: string
+          example: migration-011
+        createdAt:
+          type: string
+          example: 2020-06-04T18:49:20.672Z
+        updatedBy:
+          type: string
+          example: null
+        updatedAt:
+          type: string
+          example: 2020-06-04T18:49:20.672Z
+        statusCodes:
+            type: array
+            items:
+              type: object
+              $ref: '#/components/schemas/StatusCode'
+
+    ChangedFormDefinition:
+      type: object
+      description: Form definition updated
+      properties:
+        formId:
+          type: string
+          example: d6944dd9-1c0c-4915-971d-542dda14eb7f
+        description:
+          type: string
+          example: This is an attestation form for Agriculture and Seafood
+        createdBy:
+          type: string
+          example: migration-011
+        createdAt:
+          type: string
+          example: 2020-06-04T18:49:20.672Z
+        updatedBy:
+          type: string
+          example: migration-012
+        updatedAt:
+          type: string
+          example: 2020-06-04T18:49:20.672Z
```

Comment:
These fields are being duplicated a lot. You may want to consider using object schema inheritance to reduce duplication.
There is an example of Object inheritance/polymorphism with the Problem and ValidationError schema.
https://swagger.io/docs/specification/data-models/inheritance-and-polymorphism/

Reply:
my only issue with using inheritance is that the inherited properties show up first in the child. 
![image](https://user-images.githubusercontent.com/58443522/86042269-c4542480-b9fb-11ea-98ec-d7919dcd51ed.png)

Reply:
JSON attributes are order independent - that's not a problem.